app/helpers/events_helper.rb

```diff
@@ -0,0 +1,5 @@
+module EventsHelper
+  def creator(event)
+    User.find(event.creator_id)
```

Comment:
There should be no application logic in controllers (for instance, no `Post.new`, `Comment.create`, `current_user.friends.find_by(...)`...)

See this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png) from Pathwright